packages/inference/test/tapes.json

```diff
@@ -179,7 +179,7 @@
       "statusText": "OK"
     }
   },
-  "ea3752fcae68367386fad4918964a5de0c4eb2a46aa4854ef2ab41de13429c6b": {
+  "45d0b01267b465f464cb9e208b877ee1c7d9986fa4594e03b8b918dc04568d2c": {
```

Comment:
any ideas why all the hashes changed?

Reply:
I think it's because fetch inputs has changed to blob in these tests.